src/odb/src/zutil/util.cpp

```diff
@@ -113,20 +145,78 @@ void RUDYCalculator::calculateRUDY()
     // Iterate over the tiles in the calculated range
     for (int x = minXIndex; x <= maxXIndex; ++x) {
       for (int y = minYIndex; y <= maxYIndex; ++y) {
-        Tile& grid = getEditableTile(x, y);
-        const auto gridBox = grid.getRect();
-        if (netBox.overlaps(gridBox)) {
-          const auto intersectArea = netBox.intersect(gridBox).area();
-          const auto gridArea = gridBox.area();
-          const auto gridNetBoxRatio = static_cast<float>(intersectArea)
-                                       / static_cast<float>(gridArea);
-          const auto rudy = netCongestion * gridNetBoxRatio * 100;
-          grid.addRUDY(rudy);
+        Tile& tile = getEditableTile(x, y);
+        const auto tileBox = tile.getRect();
```

Comment:
Just a comment for future reference: variable names should use the snake case pattern, not the camel case. I know this code uses the camel case pattern everywhere, but we should avoid adding more "wrong" var names. In the future, you could refactor the var names of this code to use the correct pattern.